ui/pages/swaps/main-quote-summary/main-quote-summary.stories.js

```diff
@@ -17,18 +61,11 @@ export const DefaultStory = () => {
         padding: '24px',
       }}
     >
-      <MainQuoteSummary
-        sourceValue={text('sourceValue', '2000000000000000000')}
-        sourceDecimals={number('sourceDecimals', 18)}
-        sourceSymbol={text('sourceSymbol', 'ETH')}
-        destinationValue={text('destinationValue', '200000000000000000')}
-        destinationDecimals={number('destinationDecimals', 18)}
-        destinationSymbol={text('destinationSymbol', 'ABC')}
-        sourceIconUrl=".storybook/images/metamark.svg"
-        destinationIconUrl=".storybook/images/sai.svg"
+      <MainQuoteSummary 
+        {...args}
       />
```

Comment:
I think you will get a lint error here

```suggestion
      <MainQuoteSummary {...args} />
```